server/src/main/java/io/spine/examples/shareaware/server/wallet/WalletReplenishmentProcess.java

```diff
@@ -100,7 +101,7 @@ WalletReplenished on(BalanceRecharged e) {
                 .newBuilder()
                 .setReplenishment(state().getId())
                 .setWallet(e.getWallet())
-                .setMoneyAmount(e.getMoneyAmount())
+                .setMoneyAmount(state().getMoneyAmount())
```

Comment:
If `WalletReplenished.money_amount` means "how much money there is in the wallet after the replenishment", let's rename this field to `WalletReplenished.balance`.

Reply:
`WalletReplenished.money_amount` means "how much money the wallet should be replenished by".